test/integration/schemagic.spec.js

```diff
@@ -107,6 +106,98 @@ describe('/source/schemagic with valid example schemas', function () {
 				});
 			});
 		});
+
+		describe('validating against test1 schema that has valid foreign key value, ' +
+			'foreignKey defined with dot notation and belongs to the schema, foreignKey:true - callback', function(){
+			var result;
+			before(function(done){
```

Comment:
A whitespace bromance :-) remember to ask @bifrost alias Mr. Whitespace if he would like some extra linebreaks in there :-)